runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsConfig.java

```diff
@@ -118,4 +118,25 @@ default boolean showRespawnTimer()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		position = 8,
+		keyName = "deadNpcColor",
+		name = "Dead NPC menu entry color",
+		description = "Color of menu entries that refer to dead NPCs"
+	)
+	default Color getdeadNpcColor()
```

Comment:
You can just make this `Color getDeadNpcColor()` and it will default to no color.